src/config/constants.ts

```diff
@@ -1,3 +1,5 @@
 export const DEFAULT_CACHE_TTL = 60; // 1 minute
 export const MEDIUM_CACHE_TTL = 60 * 15; // 15 minutes
 export const LONG_CACHE_TTL = 60 * 60; // 1 hour
+
+export const DECIMAL_PLACES = 9;
```

Comment:
```suggestion
export const DEFAULT_DECIMAL_PLACES = 9;
```
I only saw it as a default value in case the actual value is not provided so I'd rename it.